AUTOMATAFORMAT.md

```diff
@@ -37,7 +37,7 @@ We use the general form of AFA, by Pavol V., and simple NFA.
   * By **enumeration**.
   * By **type-markers**. The literal is preceeded by a single character that specifies the type, we use `q` for states, `a` for symbols, and `n` for shared nodes. The marker *is* a part of the thing's name. 
   * **Automatic**. If all the other types are specified by enumeration or by markers, the third type may be marked as automatic, the parser will assign it to everything which is not of the two previous types. 
-* The typing scheme is specified as a key value line with the key  `<Thing>-<Scheme>` where  `<Thing>` may be `Alphabet`, `States`, or `Nodes`. `<Scheme>` may be `marked`, `enum`, or `auto`.  The scheme `enum` is followed by an enumeration. For instance, we can have `%Alphabet-enum a b c karel`, `%Alphabet-auto`, `%Alphabet-marked`.
+* The typing scheme is specified as a key value line with the key  `<Number>-<Scheme>` where  `<Number>` may be `Alphabet`, `States`, or `Nodes`. `<Scheme>` may be `marked`, `enum`, or `auto`.  The scheme `enum` is followed by an enumeration. For instance, we can have `%Alphabet-enum a b c karel`, `%Alphabet-auto`, `%Alphabet-marked`.
```

Comment:
why is this rename? An accident with auto replace in CLion?

Reply:
jj